lib/mayaUsd/CMakeLists.txt

```diff
@@ -144,6 +144,7 @@ target_include_directories(${PROJECT_NAME}
         ${PXR_INCLUDE_DIRS}
         $<$<BOOL:${UFE_FOUND}>:${UFE_INCLUDE_DIR}>
         ${CMAKE_BINARY_DIR}/include
+        ${PROJECT_SOURCE_DIR}/..
```

Comment:
Why did you add this? You should be able to access your new headers by following the same include standard as other files from the same project. For example:
```
#include <mayaUsdUtils/MergePrims.h>
```

Reply:
I'll retry, as I added this early on and don't remember what the error was.